README.md

```diff
@@ -50,59 +50,53 @@ This pipeline can be executed without any local installation, permissions, or cr
 
 The instructions to trigger the workflow can be found at [Sage-Bionetworks-Workflows/nf-agora](https://github.com/Sage-Bionetworks-Workflows/nf-agora)
 
+### Configuring Synapse Credentials
+
+1. Obtain download access to all required source files in Synapse, including accepting the terms of use on the AD Knowledge Portal backend [here](https://www.synapse.org/#!Synapse:syn5550378).  If you see a green unlocked lock icon, then you should be good to go.
+2. Obtain write access to the destination Synapse project, e.g. [Agora Synapse project](https://www.synapse.org/#!Synapse:syn11850457/files/)
+3. Create a Synapse personal access token (PAT)
+4. [Set up](https://help.synapse.org/docs/Client-Configuration.1985446156.html) your Synapse Python client locally
+
+Your configured Synapse credentials can be used to run this package both locally and using Docker, as outlined below.
+
 ### Locally
 Perform the following one-time steps to set up your local environment and to obtain the required Synapse permissions:
 
-1. Due to the nature of Python, you will want to set up your python environment with [conda](https://www.anaconda.com/products/distribution) or [pyenv](https://github.com/pyenv/pyenv).  You will want to create a virtual environment to do your work.
-    * conda - please follow instructions [here](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) to manage environments
-    * pyenv - you will want to use [virtualenv](https://virtualenv.pypa.io/en/latest/) to manage your python environment
+1. If you have not already, [install](https://cloud.google.com/python/docs/setup) a supported version of Python and. Versions supported by this package are all versions >3.7 and <3.11. Make sure that Python and `pip` are installed correctly and have been added to your PATH by running `python3 --version` and `pip3 --version`. If your installation was successful, your terminal will return the versions of Python and `pip` that you installed.
```

Comment:
"If you have not already, [install](https://cloud.google.com/python/docs/setup) a supported version of Python and." <-- and what? Is the "and" supposed to be there?

Reply:
the "and" was not meant to be there. Thanks for catching that!

Reply:
the "and" was not meant to be there. Thanks for catching that!